core/camel-api/src/main/java/org/apache/camel/Exchange.java

```diff
@@ -67,6 +67,7 @@
 @ConstantProvider("org.apache.camel.ExchangeConstantProvider")
 public interface Exchange extends VariableAware {
 
+    String ACTIVE_SPAN = "OpenTracing.activeSpan";
```

Comment:
qq: this is not following the pattern we use. Is there a reason for that?

Reply:
This was copied from `camel-trace`, but I agree that it should be renamed.

Reply:
This key is also used in `core/camel-support/src/main/java/org/apache/camel/saga/InMemorySagaCoordinator.java`